### PR TITLE
Adding Base Image for Papermill Docker

### PR DESCRIPTION
**Pull request by bshifaw**
This pull request introduces a Docker image that incorporates Papermill, a versatile tool for parameterizing and executing Jupyter Notebooks, alongside fundamental Python and R kernels. This addition expands our Docker image repository allowing us to execute Python and R Jupyter Notebooks within our WDL workflows. 

Changes Made
Added Dockerfile to the repository, an environment file containing a list of  Python and R packages to install, and Makefile.

### CODE REVIEW

File: docker/lr-papermill-base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
+############### stage 0
+FROM continuumio/miniconda3 AS build
+ARG PKM='conda'
+ARG CONDA_ENV_NAME='lr-papermill-base'
+ARG INSTALL_CMD="${PKM} env create -n ${CONDA_ENV_NAME}"
+ARG CLEAN_CMD="${PKM} clean --all --yes"
+
+# Install conda-pack:
```
[the remark this answers is not in the document]
Comment (SHuang-Broad) [reply docker/lr-papermill-base/Dockerfile:8]: ping to address this issue
[I'm about to use code in #438, which depends on this PR].

File: docker/lr-papermill-base/Makefile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+VERSION = 2.3.4
```
Comment (SHuang-Broad) [inline docker/lr-papermill-base/Makefile:1]: can we update this to the latest stable (2.5.0)?
my bad for delaying the review so much that you'd need to update this.

**Review by SHuang-Broad — APPROVED** [review]
Looks good, just one trivial documentation update request.

File: docker/lr-papermill-base/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (bshifaw) [reply docker/lr-papermill-base/Dockerfile:14]: done

**Pull request merged**